### PR TITLE
Added table of supported FFC camera modules

### PR DESCRIPTION
**Pull request by Erol444**
To be merged after we create "general camera" node in FW.
image

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Just a note that the * showed up as a bulleted list in the Note:
image

**Comment by Erol444** [comment]
> Just a note that the * showed up as a bulleted list in the Note:
> image

Thanks for noticing, fixed now:)

File: docs/source/pages/DM1090.rst [hunk 1]
```diff
@@ -20,6 +20,69 @@ different lenses to get **custom FoV** (they have wide and narrow angle M12 lens
...
+     - 1 MP
+     - 75° HFOV
+     - 2
+     - `Shop <https://shop.luxonis.com/addme>`__
+   * - OV9782
+     - Color
+     - 1 MP
+     - 75° HFOV
+     - 2
+     - `Shop <https://shop.luxonis.com/addme>`__
```
Comment (cafemoloko) [inline docs/source/pages/DM1090.rst:64]:
```suggestion
     - `Shop <https://shop.luxonis.com/collections/modular-cameras/products/oak-ffc-ov9782-22-pin>`__
```

Comment (cafemoloko) [inline docs/source/pages/DM1090.rst:64]:
```suggestion
     - `Shop <https://shop.luxonis.com/collections/modular-cameras/products/oak-ffc-ov9782-22-pin>`__
```

File: docs/source/pages/DM1090.rst [hunk 2]
```diff
@@ -20,6 +20,69 @@ different lenses to get **custom FoV** (they have wide and narrow angle M12 lens
...
+     - 1 MP
+     - 71.8° HFOV, 81° DFOV
+     - 2
+     - `Shop <https://shop.luxonis.com/collections/all/products/oak-ffc-ov9282-pair>`__, :ref:`docs <bg0250tg>`
+   * - IMX477
+     - Color
+     - 12 MP
+     - 75° HFOV
+     - 2*
+     - `Shop <https://shop.luxonis.com/addme>`__
```
Comment (cafemoloko) [inline docs/source/pages/DM1090.rst:52]:
```suggestion
     - `Shop <https://shop.luxonis.com/collections/modular-cameras/products/oak-ffc-imx477>`__
```

File: docs/source/pages/DM1090.rst [hunk 3]
```diff
@@ -20,6 +20,69 @@ different lenses to get **custom FoV** (they have wide and narrow angle M12 lens
...
+   * - OV9282
+     - Mono
+     - 1 MP
+     - 75° HFOV
+     - 2
+     - `Shop <https://shop.luxonis.com/addme>`__
```
Comment (cafemoloko) [inline docs/source/pages/DM1090.rst:58]:
```suggestion
     - `Shop <https://shop.luxonis.com/collections/modular-cameras/products/oak-ffc-ov9282-22-pin>`__
```

**Review by cafemoloko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**